external/Microsoft.Spark.E2ETest.ExternalLibrary/Microsoft.Spark.E2ETest.ExternalLibrary.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why not just put this project next to the unit tests?

Reply:
Didn't think it would be necessary to clutter src/csharp with this project who only satisfies one set of tests.  I noticed spark had an [external](https://github.com/apache/spark/tree/master/external) folder which had some related external code so it seemed like it'd make sense to put it here.

Reply:
moved to src/csharp